### ISSUE TITLE
About the role of MaskRefineNet 

### ISSUE THREAD

**Issue by JHang2020** [clovaai/PointWSSIS#2]
Hello! Thanks for the great work.
I am confused about that why the teacher network cannot output the "true positive" masks when the labeled data is few. The model should easily overfit to the few training data which are fully labeled and the MaskRefineNet hardly needs to make any changes to achieve good training set performance. So what does the MaskRefineNet learn in this case?

 Thanks a lot if you can help answer my puzzles!

**Comment by qjadud1994** [comment]
Thanks for the insightful comments.

As you mentioned, the teacher network tends to suffer the overfitting issue when the amount of fully-labeled data is extremely limited, and the MaskRefineNet is proposed to alleviate this limitation.

We think the effectiveness of MaskRefineNet can be found for the following reasons.

(1) We assume we have N fully-labeled data, and each data contains M instances. The training data for the MaskRefineNet can be augmented M times (teacher network: N training samples, MaskRefineNet: N*M training samples).

(2) Thanks to helpful knowledge from the teacher network (i.e., rough mask) and explicit instance guiding using point labels, we can reduce the optimization space for training the MaskRefineNet.
